src/app/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.datasource.jdbc-url=jdbc:postgresql://localhost/hint
```

Comment:
Use this configuration in  tests, where the user CLI classes are not running within a container, hence can't access the hint db by the container name, needs to go to localhost instead,

Reply:
I guess this means the value set below in `application.properties` won't work for local development either. We'll have to figure out how to define one set of properties for dev, another for prod. Looks like there's some relevant documentation here: https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html

Reply:
We will - I thought we could cross that bridge when we put the Pac4 Db authentication into login in the next sprint...